### PR TITLE
Fork windows image build for staging builds

### PR DESCRIPTION
**Pull request by jjerger**
Enables build and publish to testing for images using the staging Rapture repo. Stripped all publishing tasks to prod and internal for safety.

### CODE REVIEW

File: concourse/pipelines/windows-image-build-staging.jsonnet [hunk 1]
```diff
@@ -557,7 +557,7 @@ local ImgGroup(name, images) = {
   name: name,
   env: 'testing',
   jobs: [
-    'build-' + image
+    'build-' + image + '-staging',
     for image in images
   ] + [
     'publish-to-staging-%s' % [image]
```
Comment (bkatyl) [inline concourse/pipelines/windows-image-build-staging.jsonnet:563]: I believe this should be 'publish-to-testing-%s'

File: concourse/pipelines/windows-image-build-staging.jsonnet [hunk 2]
```diff
@@ -548,7 +548,7 @@ local MediaImgPublishJob(image, workflow_dir, gcs_dir) = imgpublishjob {
   image: image,
   gcs_dir: gcs_dir,
   // build -> testing
-  passed:: 'build-' + image,
+  passed:: 'build-' + image + '-staging',
```
Comment (bkatyl) [inline concourse/pipelines/windows-image-build-staging.jsonnet:551]: I believe -staging --> -testing

File: concourse/pipelines/windows-image-build-staging.jsonnet [hunk 3]
```diff
@@ -539,7 +539,7 @@ local ImgPublishJob(image, workflow_dir, gcs_dir) = imgpublishjob {
   env: 'testing',
   gcs_dir: gcs_dir,
   // build -> testing
-  passed:: 'build-' + image,
+  passed:: 'build-' + image + '-staging',
```
Comment (bkatyl) [inline concourse/pipelines/windows-image-build-staging.jsonnet:542]: I believe -staging --> -testing

File: concourse/pipelines/windows-image-build-staging.jsonnet [hunk 4]
```diff
@@ -557,7 +557,7 @@ local ImgGroup(name, images) = {
   name: name,
   env: 'testing',
   jobs: [
-    'build-' + image
+    'build-' + image + '-staging',
```
Comment (bkatyl) [inline concourse/pipelines/windows-image-build-staging.jsonnet:560]: I believe -staging --> -testing

**Comment by dorileo** [comment]
even though we are splitting staging and non staging, can we share common components? I'm wondering if we can reuse stuff and have the separation more from a execution environment POV than the "descriptors" existence.

**Comment by bkatyl** [comment]
This fork is to use the staging repos. Refactoring the templates so there is more re-use should be more of a long term goal. We were trying to do this quickly to be able to get enough test coverage for the guest agent changes so we can release the package updates. 

Currently we have no automated way of testing packages in our unstable or staging repos.

**Review by bkatyl — APPROVED** [review]

**Pull request merged**